lib/google_distance_matrix/client.rb

```diff
@@ -15,18 +15,27 @@ class Client
     #
     # @param url              The URL to Google's API we'll make a request to
     # @param instrumentation  A hash with instrumentation payload
-    # @param options          Other options we don't care about, for example we'll capture
-    #                         `configuration` option which we are not using, but the ClientCache
-    #                         is using.
+    # @param configuration    Instance of Configuration telling us what timeout
+    #                         to use, for instance
     #
     # @return Hash with data from parsed response body
-    def get(url, instrumentation: {}, **_options)
+    def get(url, instrumentation: {}, configuration: nil)
```

Comment:
Don't think we get the configuration directly here, but a hash which contains `:configuration`

Called from:

https://github.com/Skalar/google_distance_matrix/blob/3b1f9982d8741e7db5712a555fece0363c445a7a/lib/google_distance_matrix/matrix.rb#L117-L121

Used:
https://github.com/Skalar/google_distance_matrix/blob/3b1f9982d8741e7db5712a555fece0363c445a7a/lib/google_distance_matrix/client_cache.rb#L21

So, I'm a bit confused here and not sure if it works? Therefor I don't think this change to `**_options` and it's documentation is correct.